eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/model/eva/SubmittedVariantEntity.java

```diff
@@ -64,9 +66,18 @@
     @Field("validated")
     private Boolean validated;
 
+    private String remappedFrom;
+
     protected SubmittedVariantEntity() {
     }
 
+    //Constructor to be used to store remapped submitted variants
+    public SubmittedVariantEntity(Long accession, String hashedMessage, ISubmittedVariant model, int version,
+                                  String remappedFrom) {
+        this(accession, hashedMessage, model, version);
+        this.remappedFrom = remappedFrom;
```

Comment:
This is more of an enhancement suggestion (that can be handled in a separate ticket). Should we also have a re-mapping metadata (unique ID for a given remap run, from/to assemblies, timestamp etc.,)  in 
a separate collection somewhere and refer to that unique ID from within SVE? We can still retain remappedFrom in SVE for the sake of querying efficiency. Such cross-references could come in handy for troubleshooting.

Reply:
I think it makes sense to implement it but I agree it is better to do it in another ticket, this getting too big in scope. Added this in ticket [EVA-2420](https://www.ebi.ac.uk/panda/jira/browse/EVA-2420)